AmazonChimeSDK/AmazonChimeSDK/internal/video/DefaultVideoClientController.swift

```diff
@@ -287,7 +287,7 @@ extension DefaultVideoClientController: VideoClientDelegate {
         return uris.map(self.configuration.urlRewriter)
     }
 
-    public func videoClientDidCompletePrimaryMeetingJoin(_ status: video_client_status_t) {
+    public func videoClientDidPromote(toPrimaryMeeting status: video_client_status_t) {
```

Comment:
why doesn't it look like `videoClientDidPromotetoPrimaryMeeting(status: video_client_status_t)`?